docs/open-source/_toc.json

```diff
@@ -0,0 +1,40 @@
+{
+  "title": "Open source",
+  "collapsed": true,
+  "children": [
+    {
+      "title": "Code of conduct",
+      "url": "/open-source/code-of-conduct"
+    },
+    {
+      "title": "Contributor license agreements",
+      "children": [
+        {
+          "title": "Individual contributor",
+          "url": "https://docs.quantum.ibm.com/open-source/qiskit-cla.pdf"
+        },
+        {
+          "title": "Corporate contributor",
+          "url": "https://docs.quantum.ibm.com/open-source/qiskit-corporate-cla.pdf"
+        }
+      ]
+    },
+    {
+      "title": "Qiskit SDK",
+      "children": [
+        {
+          "title": "Version strategy",
```

Comment:
This is aka the deprecation policy. It explains what the policy is, whereas the DEPRECATION.md file in qiskit repo explains the tactics of how maintainers write deprecations.

Reply:
I dont see the deprecation policy line in this file? Or is the diff just bad? I just see version strategy then contributor guide

Reply:
We were thinking the deprecation guide should be linked inside the "Version strategy" file, rather than having a dedicated external link in the left ToC. What do you prefer? We can add an external link like "Deprecation techniques" or "Deprecation guide"

Reply:
ah ok yeah keeping it in the version strategy page is fine by me 👍